safe_control_gym/controllers/mpc/linear_mpc.py

```diff
@@ -261,19 +254,20 @@ def select_action(self,
             self.u_prev = u_val
             self.results_dict['horizon_states'].append(deepcopy(self.x_prev) + self.X_EQ[:, None])
             self.results_dict['horizon_inputs'].append(deepcopy(self.u_prev) + self.U_EQ[:, None])
-        except RuntimeError as e:
-            print(e)
+        except RuntimeError:
```

Comment:
why not print out the error `e`?

Reply:
Thanks for pointing this out. This will be reverted.